### PR TITLE
Refactored check of configured FirewallSSO to pf::firewallsso

### PR DESCRIPTION
**Pull request by jrouzierinverse**

### CODE REVIEW

File: lib/pf/firewallsso.pm [hunk 1]
```diff
@@ -2,13 +2,13 @@ package pf::firewallsso;
 
 =head1 NAME
 
-pf::web::firewallsso
+pf::firewallsso
 
 =cut
 
 =head1 DESCRIPTION
```
Comment (dwlfrth) [inline lib/pf/firewallsso.pm:9]: Can we document the module rather than repeating it's name ?

File: sbin/pfdhcplistener [hunk 2]
```diff
@@ -359,8 +359,8 @@ sub parse_dhcp_request {
         );
         update_iplog( $dhcp->{'chaddr'}, $client_ip, $lease_length );
         # if there is at least a firewall sso defined in the config && we have the lease length
```
Comment (dwlfrth) [inline sbin/pfdhcplistener]: Comment is no longer accurate

File: lib/pf/firewallsso.pm [hunk 3]
```diff
@@ -42,6 +43,7 @@ Send the firewall sso update request to the webapi.
 
 sub do_sso {
```
Comment (dwlfrth) [inline lib/pf/firewallsso.pm:46]: Since we are in there, can we add some logging to the code ?

**Comment by jrouzierinverse** [comment]
Please review

**Comment by fdurand** [comment]
Except Derek´s comment it´s ok for me

**Comment by jrouzierinverse** [comment]
I updated the pod doc as Derek suggested please let me know if there is anything else.

**Pull request merged**